web3.md

```diff
@@ -1810,6 +1809,50 @@ Mining
          <receiptsRoot> _ => #parseHexWord( Keccak256( #rlpEncodeMerkleTree( #receiptsRoot ) ) ) </receiptsRoot>
 ```
 
+Retrieving logs
+---------------
+
+ - `LogData` contains:
+    - a List of log elements like `{ ACCT | TOPICS:List | DATA }`
+    - Transaction Index (Int)
+    - Transaction Hash (String)
+    - Block Number (Int)
+    - Block Hash (String)
+
+[TODO]:
+ - remove hardcoded BlockHash and retrieve the correct value
+ - add rules for the cases in which `START == 'pending'` or `END == 'pending'`
+
+```k
+
+    syntax LogData ::= "{" List "|" Int "|" String "|" Int "|" String "}"
+ // ---------------------------------------------------------------------
+    
+    syntax List ::= #getLogDataBetween ( BlockIdentifier, BlockIdentifier ) [function]
+                  | #getLogDataBetweenAux ( BlockIdentifier, BlockIdentifier, List ) [function]
+ // --------------------------------------------------------------------------------------------
+    rule #getLogDataBetween(START, END) => #getLogDataBetweenAux(START, END, .List)
+
+    rule [[ #getLogDataBetweenAux(START:Int, END:Int, RESULT) => #getLogDataBetweenAux(START +Int 1, END, (RESULT ListItem({LOGS|getIndexOf(TXID, TXLIST)|TXHASH|START|"0x0"}))) ]]
+       <txReceipt>
+         <txBlockNumber>   START </txBlockNumber>
+         <txHash>          TXHASH </txHash>
+         <txID>            TXID </txID>
+         <logSet>          LOGS </logSet>
+         ...
+       </txReceipt>
+       <txOrder> TXLIST </txOrder>
+
+
+    rule #getLogDataBetweenAux(START:Int, END:Int, RESULT) => RESULT
+       requires START >Int END
+
+    syntax KItem ::= "#eth_getLogs"
+ // -------------------------------
+    rule <k> #eth_getLogs => #rpcResponseUnimplemented ... </k>
```

Comment:
If we do (something like)

```
    rule <k> #eth_getLogs => #getLogData(#parseBlockIdentifier(P1), #parseBlockIdentifier(P2), TXRECEIPTS) ... </k>
         <txReceipts> TXRECEIPTS </txReceipts>
         <params> [ P1 , P2 , .JSONs ] </params>
```

we can avoid introducing new contextual functions.